thread_pool.h

```diff
@@ -16,7 +16,7 @@
 #ifndef _GNU_SOURCE
 #define _GNU_SOURCE
 #endif
-#if HAVE_CPU_SET_T
+#ifdef HAVE_CPU_SET_T
```

Comment:
You don't need this change anymore

Reply:
I made a mistake with replacing the commit. It should be fixed now.

I was wondering though, is there a reason why `#cmakedefine01` isn't used for anything else?